### PR TITLE
[Merged by Bors] - Update Prater ENRs

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

Update bootnodes for Prater. There are new IP addresses for the Sigma Prime nodes. Teku and Nimbus nodes were also added.

## Additional Info

Related: https://github.com/eth-clients/goerli/commit/24760cd4b46c4f2274cf333375bfbf6133f44401

### CODE REVIEW

File: common/eth2_network_config/built_in_network_configs/prater/boot_enr.yaml [hunk 1]
```diff
@@ -7,4 +7,6 @@
 # Prysm bootnode #1
 - enr:-Ku4QFmUkNp0g9bsLX2PfVeIyT-9WO-PZlrqZBNtEyofOOfLMScDjaTzGxIb1Ns9Wo5Pm_8nlq-SZwcQfTH2cgO-s88Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDkvpOTAAAQIP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQLV_jMOIxKbjHFKgrkFvwDvpexo6Nd58TK5k7ss4Vt0IoN1ZHCCG1g
 # Lighthouse bootnode #1
-- enr:-LK4QLINdtobGquK7jukLDAKmsrH2ZuHM4k0TklY5jDTD4ZgfxR9weZmo5Jwu81hlKu3qPAvk24xHGBDjYs4o8f1gZ0Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpB53wQoAAAQIP__________gmlkgnY0gmlwhDRN_P6Jc2VjcDI1NmsxoQJuNujTgsJUHUgVZML3pzrtgNtYg7rQ4K1tkWERgl0DdoN0Y3CCIyiDdWRwgiMo
+- enr:-Ly4QFPk-cTMxZ3jWTafiNblEZkQIXGF2aVzCIGW0uHp6KaEAvBMoctE8S7YU0qZtuS7By0AA4YMfKoN9ls_GJRccVpFh2F0dG5ldHOI__________-EZXRoMpCC9KcrAgAQIIS2AQAAAAAAgmlkgnY0gmlwhKh3joWJc2VjcDI1NmsxoQKrxz8M1IHwJqRIpDqdVW_U1PeixMW5SfnBD-8idYIQrIhzeW5jbmV0cw-DdGNwgiMog3VkcIIjKA
```
Comment (michaelsproul) [inline common/eth2_network_config/built_in_network_configs/prater/boot_enr.yaml:10]: I think it probably doesn't make any difference, but these ENRs have Altair fork digests and the actual nodes have forked to Bellatrix

```
Eth2 Field:
	Fork digest: 82f4a72b
	Next fork version: 02001020
	Next fork epoch: 112260
	SSZ Bytes: 82f4a72b0200102084b6010000000000
```

The Bellatrix fork version is `c2ce3aa8`

Comment (pawanjay176) [reply common/eth2_network_config/built_in_network_configs/prater/boot_enr.yaml:10]: I think this is okay. We will use this node to find other nodes that has bellatrix fork digests, we just won't return nodes with bellatrix fork digests in a discovery requests. The relevant logic is here https://github.com/sigp/lighthouse/blob/df51a73272489fe154bd10995c96199062b6c3f7/beacon_node/lighthouse_network/src/discovery/mod.rs#L760-L767

**Comment by paulhauner** [comment]
I've just added some Teku and Nimbus bootnodes from https://github.com/eth-clients/eth2-networks/blob/274e71c7af8fb26f65b47016ffa6169079315e2c/shared/prater/bootstrap_nodes.txt.

**Comment by paulhauner** [comment]
I compiled this, dropped my database and confirmed that I can find peers with this list of boot ENRs.

**Review by michaelsproul — APPROVED** [review]
Batching this

**Pull request closed without merging**